### PR TITLE
Update common-planning-a-mission-with-waypoints-and-events.rst

### PR DESCRIPTION
**Pull request by auturgy**
Add zeroth waypoint / home position warning.
Link to https://github.com/ArduPilot/ardupilot/issues/1379 and https://github.com/ArduPilot/ardupilot_wiki/issues/867

### CODE REVIEW

**Comment by WickedShell** [comment]
I don't think this is correct. While the behavior specified is not wrong, that is a GCS handling item. Warning users of it will only create confusion and problems. What GCS handled this incorrectly for you?

**Comment by WickedShell** [comment]
To clarify more, I think this deserves documentation in a developer part of the wiki, or the MAVLink docs, but it doesn't belong in the end user facing documentation.

**Comment by auturgy** [comment]
I'm just clearing out issues: but missionplanner definitely does this, and as I understand things it is the intended behavior for gcs software to do so (set home to waypoint zero).
I don't see it as "incorrect", just something to be aware of.
The referenced issues provide more detail, but enough people have made this mistake that I think the note is warranted. It's up to you: but if you reject the pr, you should also close the referenced issues.

Regards,

James

File: common/source/docs/common-planning-a-mission-with-waypoints-and-events.rst [hunk 1]
```diff
@@ -18,16 +18,22 @@ This page describes generic waypoint setup for all types of vehicles.
...
 
-For **Plane** the home position is the location of the plane where the
-GPS was locked. This means if you execute an RTL in Plane, it will
-return to the location where it first acquired the GPS lock, so only
-power up your plane in the location you intend it to return to when
-performing an RTL.
+   When an auto mission is executed, the first waypoint in the mission will
+   be interpretted as the home position, overwriting any other home position
+   that is set. This means that users **MUST** set the first waypoint at 
+   their desired home position to ensure safe Take-Off and RTL behaviour.
```
Comment (OXINARF) [inline common/source/docs/common-planning-a-mission-with-waypoints-and-events.rst]: I'm pretty sure this is incorrect. The first waypoint is indeed home, but it isn't used for anything. So the user doesn't have to set it to anything and it certainly doesn't overwrite the set home position.
If we want to put a warning, it should be a dev warning, saying that in the MAVLink protocol waypoint number 0 is reserved to be the home location for legacy purposes (but it shouldn't be used to retrieve home anymore).

**Comment by magicrub** [comment]
This was a bug in Mission Planner that was fixed via https://github.com/ArduPilot/MissionPlanner/commit/97bdafb3c570fa93bf5aa189b81343a08b7798d9
closing...

**Comment by auturgy** [comment]
Thanks Tom

Regards,

James

**Pull request closed without merging**